examples/braket_features/Native_gates_on_IonQ.ipynb

```diff
@@ -0,0 +1,1666 @@
+{
```

Comment:
<p>To me this giant matrix is really much harder to read then the code itself. If we want to provide a nicer way to view it, should we just embed an image instead?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/309/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
I really like the sympy/mathjax printing, it's much easier for me to compare with textbook equations. I know in NBreview it renders as ascii instead of mathjax, was that the issue?